src/web/_market/portal/meta.json

```diff
@@ -11,39 +11,37 @@
   "versionDisplay": "portal",
   "installMatcher": "best-match",
   "mavenArtifacts": [
-    {
-      "key": "portal-10",
-      "name": "Portal App (10.0)",
-      "groupId": "ch.ivyteam.ivy.project.portal",
-      "artifactId": "portal-app",
-      "type": "zip"
-    },
-    {
-      "name": "Portal Guide (10.0)",
-      "groupId": "ch.ivyteam.ivy.project.portal",
-      "artifactId": "portal-guide",
-      "type" : "zip",
-      "doc": true
-    },
     {
       "key": "portal",
       "name": "Portal App",
       "groupId": "com.axonivy.portal",
+      "archivedArtifact": [
+        {
+          "groupId": "ch.ivyteam.ivy.project.portal",
+          "version": "10.0.19"
```

Comment:
now you have here a specific version? what happens when 10.0.20 will be released?

Reply:
I think we should not have fix versions here, only have the coordinates of an artifact.

groupId, artifactId

Reply:
Hi @alexsuter 
I handled all of your feedback except this one 😄 
Is there any potential issue that you worry about when we defined the last version in the archived artifact?
If I remove it, it's impossible to identify the correct groupId or artifactId that should be used to build the download URL when we combine Portal App (10.0) and Portal App.

![image](https://github.com/axonivy-market/market.axonivy.com/assets/127725498/4e119263-6eef-42ef-9dae-de4850584858)
I tested the downloading function and it works for the old version, even if we defined 2 or more archived artifacts.

Reply:
Then we should rename it to 'lastVersion'